app/assets/javascripts/application/open_all.js.coffee

```diff
@@ -0,0 +1,9 @@
+
+jQuery ->
+$("#open_all").click ->
+  import_url_list = $(this).data('bgeigie');
+  if import_url_list != -1
```

Comment:
I am not sure if this is JavaScript-way or not. Why not checking if `import_url_list` is empty like `import_url_list.length == 0` (and keep `unmoderated_id_list` helper as before)?

Reply:
well, sometimes I overthink a little bit and it gets more complicated than it needed to be ;)